site/docs/v1/tech/admin-guide/releases.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: doc
+title: Releases
+description: How can you see when a new version of FusionAuth is released?
+navcategory: admin
+---
+
+== Overview
+
+FusionAuth is a single tenant SaaS or downloadable solution. This fact means that you control when upgrades occur, rather than having an upgrade with new functionality and possibly breaking changes forced on you and your team.
+
+[NOTE.note]
+====
+Authentication and authorization are core dependencies of your application, and we hate forced dependency upgrades.
+
+You and you alone control when you upgrade.
+====
+
+* <<Release Notifications>>
+* <<Early Access Program>>
+
+== Release Notifications
+
+include::docs/v1/tech/shared/_release-notification.adoc[]
+
+== Early Access Program
+
+[NOTE.since]
+====
+Available since 1.41.0
+====
+
+FusionAuth has an early access program (EAP).
+
+The EAP allows you, within certain constraints, to test drive the features and functionality of the next release of FusionAuth. 
+
+=== Constraints
+
+The EAP constraints include:
+
+* You can install FusionAuth via zip, deb or rpm files. These will be available on link:/direct-download[the direct download page].
+* There is no supported upgrade path from an EAP version to a released version.
+* The performance and stability of an EAP FusionAuth instance is not guaranteed.
+* Documentation for EAP features may be lacking.
+* EAP versions are available in FusionAuth Cloud only for Essentials and Enterprise customers.
+
+[WARNING.warning]
+====
+There is no guaranteed upgrade path from an EAP version to a released version.
+
+It should be used only to test features and integrations in order to prepare for an upcoming release.
+====
+
+While the FusionAuth team will make every effort to release an EAP version with stable APIs, they may change. In addition, there is no supported upgrade path from an EAP instance to a released version.
+
+=== Usage Guidelines
+
+An EAP is appropriate for the following purposes:
+
+* testing out features
+* verifying a new FusionAuth version won't break your existing integrations
+* load testing to gain performance insights
+* running end-to-end tests against to ensure your applications can be safely upgraded
+* migrating a production dataset in a test environment to quantify upgrade durations
+* giving feedback to the FusionAuth team
+
+You **should not use** an EAP to:
+
+* authenticate production users
+
+[WARNING.warning]
+====
+There is no guaranteed upgrade path from an EAP version to a released version.
```

Comment:
Remove guaranteed- I think we should say - there is not an upgrade path, it is a dead end.